dataactcore/scripts/databaseSetup.py

```
@@ -0,0 +1,30 @@
+import sqlalchemy
+from sqlalchemy.exc import ProgrammingError, IntegrityError
+
+def runCommands(credDict, sqlCommands, dbName, connection = None):
+    """ Apply commands to specified database """
+    dbBaseName = "postgres"
+    baseEngine = sqlalchemy.create_engine("postgresql://"+credDict["username"]+":"+credDict["password"]+"@"+credDict["host"]+":"+credDict["port"]+"/"+dbBaseName, isolation_level = "AUTOCOMMIT")
+
+    try:
+        connect = baseEngine.connect()
+        rows  = connect.execute("SELECT 1 FROM pg_database WHERE datname = '" +dbName+"'")
+        if ((rows.rowcount) == 0) :
+            connect.execute("CREATE DATABASE " + '"' + dbName + '"')
+        connect.close()
+    except ProgrammingError as e:
+        # Happens if DB exists, just print and carry on
+        # print(e.message)
```

should log the error instead of failing silently